data/tlg0018/tlg001/tlg0018.tlg001.1st1K-eng1.xml

```diff
@@ -0,0 +1,1013 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">The Works of Philo Judaeus</title>
+<sponsor>University of Leipzig</sponsor>
+<funder>European Social Fund Saxony</funder>
+<principal xml:id="GRC">Gregory Crane</principal>
+<respStmt>
+<persName xml:id="Jouve">Jouve</persName>
+<resp>OCR-ed, did a first pass of correction on, and encoded the text</resp>
+</respStmt>
+<respStmt>
+<persName>Gregory Crane</persName>
+<resp>In 2023, GRC broke the translations up into works, cleaned up tagging of notes, and used digital files from CCEL.org to follow the sections as the appear in the David Scholer edition of the Yonge translation. He corrected many errors in the text but many remain.</resp>
+</respStmt>
+<respStmt>
+<persName>Greta Franzini</persName>
+<resp>Project Manager (University of Leipzig)</resp>
+</respStmt>
+<respStmt>
+<persName>Bruce Robertson</persName>
+<resp>Technical Advisor (Mount Allison University)</resp>
+</respStmt>
+</titleStmt>
+<publicationStmt>
+<authority>University of Leipzig</authority>
+<idno type="filename">philo_1_1854.xml</idno>
+<availability>
+<licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+</availability>
+<date>2014</date>
+<publisher>University of Leipzig</publisher>
+<pubPlace>Germany</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl xml:lang="la">
+<biblStruct>
+<monogr>
+<editor>
+<persName>
+<name xml:lang="en">Charles Duke Yonge</name>
+</persName>
+</editor>
+<author>Philo Alexandrinus</author>
+<title xml:lang="eng">On the Creation of the World</title>
+<title xml:lang="eng">On the Alegories of the Sacred Laws</title>
+<title xml:lang="eng">On the Cherubim; and on the Flaming Sword; and on the First-Born Child of Man, Cain</title>
+<title xml:lang="eng">Of Cain and his Birth</title>
+<title xml:lang="eng">On the Sacrifice of Abel and Cain</title>
+<title xml:lang="eng">On the Principle that the EWorse is accustomed to be always plotting against the Better</title>
+<title xml:lang="eng">On the Posterity of Cain</title>
+<title xml:lang="eng">On the Giants</title>
+<title xml:lang="eng">On the Unchangeableness of God</title>
+<title xml:lang="eng">On the Tilling of the Earth by Noah</title>
+<title xml:lang="eng">About the Planting of Noah</title>
+<title xml:lang="eng">On Drunkenness</title>
+<title xml:lang="eng">On Sobriety</title>
+<imprint>
+<publisher>George Bell and Sons</publisher>
+<pubPlace>London</pubPlace>
+<date>1854</date>
+</imprint>
+<biblScope unit="volume">1</biblScope>
+</monogr>
+ <ref
+target="https://archive.org/details/worksphilojudaeu01philuoft" >Internet Archive</ref>
```

Comment:
We've been updating the links here to go to the exact page a work starts on rather than the top level page so in this case you would change it to: 
https://archive.org/details/worksphilojudaeu01philuoft/page/n13/mode/2up

Reply:
This is a cool feature. I have the pages in now.